objects/src/accounts/component/template/storage/mod.rs

```diff
@@ -315,8 +315,9 @@ impl Deserializable for StorageEntry {
 // ================================================================================================
 
 /// Represents the type of values that can be found in a storage slot's `values` field.
-#[cfg_attr(feature = "std", derive(serde::Deserialize, serde::Serialize))]
-#[cfg_attr(feature = "std", serde(untagged))]
+#[cfg(feature = "std")]
+#[derive(serde::Deserialize, serde::Serialize)]
+#[serde(untagged)]
 enum StorageValues {
```

Comment:
It seems like this struct (and its impl) actually should go to the `src/accounts/component/template/storage/toml.rs` file (and then gating it by `std` feature would be unnecessary). Let's move it there.